patches/minecraft/net/minecraft/world/entity/projectile/AbstractArrow.java.patch

```diff
@@ -9,14 +9,36 @@
           this.m_20095_();
        }
  
-@@ -186,7 +_,9 @@
+@@ -171,7 +_,7 @@
+             vec33 = hitresult.m_82450_();
+          }
+ 
+-         while(!this.m_213877_()) {
++         loop: while(!this.m_213877_()) {
```

Comment:
Instead of modifying the line, the label can instead be placed on the line before the loop. Also, if possible, could you please give a more descriptive name than `loop` here?